docs/source/models/cause.rst

```diff
@@ -4,66 +4,108 @@
+    1. What components are used to implement the cause model?
+    2. How do you include them in the simulation?
+    3. What columnsdo they create in the population table?  What do the columns mean?
+    4. What value pipelines do they create? What do the values coming out of 
+       the pipelines mean?
+- **Parameters:** List all available parameters in the format of *parameter name* plus
+  *parameter description*. This section is primarily necessary for non standard 
+  components. It should otherwise point to standard documentation or state that 
+  no parameters are available.
+
 
 Common cause models
 -------------------
 
-.. todo::
-
-   Format as table with model type, description.
-   Fill in descriptions.
+**SI** - Short for susceptible-infected. This modeling strategy is appropriate for 
+causes where people never recover once they enter into a with condition state. 
+Example: Ischemic Heart Disease
 
-SI
-++
+**SIS** - Short for susceptible-infected-susceptible. This modeling strategy 
+is appropriate for causes where individuals can have multiple cases over 
+the course of their lives. Example: Diarrheal Diseases
 
-SIS
-+++
+**SIR** - Short for susceptible-infected-recovered. This modeling strategy
+is appropriate for causes where individuals can only have a single
+case, but do not stay in the with condition state forever. Example: Measles
 
-SIR
-+++
-
-Neonatal
-++++++++
+**Neonatal** - This modeling strategy is appropriate for causes that an 
+individual is born into the with condition state with and never recovers from.
+Example: Preterm birth
 
 Common data sources for cause models
 ------------------------------------
 
-.. todo::
-
-   Format as table with measure, measure definition, data sources and
-   their uses.
-
-Incidence
-+++++++++
-
-Birth prevalence
-++++++++++++++++
-
-Remission
-+++++++++
-
-Prevalence
-++++++++++
-
-Cause-specific mortality
-++++++++++++++++++++++++
-
-Excess mortality
-++++++++++++++++
-
-Disability weight
-+++++++++++++++++
+.. note::
+
+   Currently there are GBD shared function tools that return you draw-level values;
+   alternatively, you can also pull data by using ``vivarium_inputs`` tools.
+
+Available Tools
+
++------------------+-------------------------------------------------------+
+| **Tool**         | **Requirements**                                      |
++------------------+-------------------------------------------------------+
+| get_draw("como") | only accepts one of the following types:              |
+|                  | cause_id, rei_id, or sequela_id                       |
++------------------+-------------------------------------------------------+
+| get_draw("epi")  | with rare exceptions, Prevalence, Incidence, EMR, and |
+|                  | WCMR values are non-final and should not be used      |  
++------------------+-------------------------------------------------------+
+| get_raw_data()   | modelable entity, measure, and lcoation               |
```

Comment:
typo in `location` :D

Reply:
Thanks, James introduced me the reStructuredText Directives that includes a lot of contents such as how to make a table using "list-table". I will definitely make those documents easily editable as i learning through rst markdowns.